var/spack/repos/builtin/packages/netlib-lapack/package.py

```diff
@@ -149,13 +150,16 @@ def install_one(self, spec, prefix, shared):
             cmake_args.extend(['-DCBLAS=OFF'])
             cmake_args.extend(['-DLAPACKE:BOOL=OFF'])
 
-        if self.compiler.name == 'xl' or self.compiler.name == 'xl_r':
-            # use F77 compiler if IBM XL
-            cmake_args.extend([
-                '-DCMAKE_Fortran_COMPILER=%s' % self.compiler.f77,
-                '-DCMAKE_Fortran_FLAGS=%s' % (
-                    ' '.join(self.spec.compiler_flags['fflags'])),
-            ])
+        if spec.satisfies('arch=linux-rhel7-ppc64le'):
```

Comment:
Surely XL are not only used here? I think u need to rework this ‘if’.

Ps.i don’t know anything about XL either.

Reply:
@davydden Do you know anyone that you could suggest?